packages/ag-charts-website/src/content/docs/charts-nightingale-series/index.mdoc

```diff
@@ -0,0 +1,145 @@
+---
+title: "Nightingale Series"
+enterprise: true
+---
+
+**Nightingale** series (sometimes called a *Rose chart*) represent a dataset in a form of a polar area diagram.
```

Comment:
A Nightingale series, also known as a Rose chart, represent**_s_** a dataset in **_the_** form of a polar area diagram.